app/models/page_widget.rb

```diff
@@ -8,6 +8,6 @@
 #
 
 class PageWidget < ActiveRecord::Base
-  belongs_to :page
+  belongs_to :page, dependent: :destroy
```

Comment:
This won't be really needed cause we don't use PageWidget anymore (I'm removing it in the refactor).
But let it be for now